cypress/integration/models/stakeholdergroups.ts

```diff
@@ -15,34 +26,47 @@ export class Stakeholdergroups {
     }
 
     protected fillName(name: string): void {
-        inputText(stakeholdergroupName, name);
+        inputText(stakeholdergroupNameInput, name);
     }
 
     protected fillDescription(description: string): void {
-        inputText(stakeholdergroupDescription, description);
+        inputText(stakeholdergroupDescriptionInput, description);
+    }
+
+    protected selectMember(member: string): void {
+        selectFormItems(stakeholdergroupMemberSelect, member);
     }
 
     getStakeholdergroupName(): string {
         this.stakeholdergroupName = faker.company.companyName();
         return this.stakeholdergroupName;
     }
 
+    getNewStakeholdergroupName(): string {
+        return faker.company.companyName();
+    }
+
     getStakeholdergroupDescription(): string {
         this.stakeholdergroupDescription = faker.lorem.sentence();
         return this.stakeholdergroupDescription;
     }
 
-    create(): void {
+    create(name?: string, description?: string, member?: string): void {
         Stakeholdergroups.clickStakeholdergroups();
         clickByText("button", "Create new");
         this.getStakeholdergroupName();
         this.getStakeholdergroupDescription();
+        this.stakeholdergroupName = name || this.stakeholdergroupName;
         this.fillName(this.stakeholdergroupName);
```

Comment:
Why can't we do 


              this.fillName(faker.company.companyName())
and


              this.fillDescription(faker.lorem.sentence());

Reply:
if we do this then we will not get `this.stakeholdergroupName` variable initiazed. which will be useful in case of edit and delete. Otherwise we would always need to send this name to edit and delete.